fvm/transaction.go

```diff
@@ -77,6 +78,8 @@ func (proc *TransactionProcedure) Run(vm *VirtualMachine, ctx Context, st *state
 		}
 	}
 
+	proc.MemoryUsed = uint64(st.State().TotalMemoryUsed())
```

Comment:
can you move this to `transactionInvoker` and duplicate the logic for computation used:
- https://github.com/onflow/flow-go/blob/8f7942581efe7d86f7f7121a7cb6c1daf9fd3499/fvm/transactionInvoker.go#L248
- https://github.com/onflow/flow-go/blob/8f7942581efe7d86f7f7121a7cb6c1daf9fd3499/fvm/transactionInvoker.go#L167

That way we have that logic together.